my-bundle/src/main/gateway/config/services.yml

```diff
@@ -0,0 +1,9 @@
+my-bundle/testservice:
```

Comment:
Service name has to be the exact name of the service in the my-bundle.bundle file.
Otherwise it won't match.

Reply:
hmmm.. gotcha..